doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,106 @@
+larger and larger. The area (size) values are available in a pandas-compatible
+format, which makes for convenient data analysis and visualization.
+
+Beyond area, many other region properties are available.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from skimage import data, measure
+
+
+fractions = np.linspace(0.05, 0.5, 10)
+
+#####################################################################
+# 2D images
+# =========
+
+images = [data.binary_blobs(volume_fraction=f) for f in fractions]
+
+labeled_images = [measure.label(image) for image in images]
+
+properties = ['label', 'area']
+
+tables = [measure.regionprops_table(image, properties=properties)
+          for image in labeled_images]
+tables = [pd.DataFrame(table) for table in tables]
+
+for fraction, table in zip(fractions, tables):
+    table['volume fraction'] = fraction
+
+areas = pd.concat(tables, axis=0)
+
+# Divide the figure into a 2x1 grid
+fig, (im, ax) = plt.subplots(2, 1)
+# Turn off axes for top subplot
+im.axis('off')
+# Plot area vs volume fraction
+areas.plot(x='volume fraction', y='area', kind='scatter', ax=ax)
+# Show image with lowest volume fraction
+ax1 = fig.add_subplot(221)
+ax1.imshow(images[0], cmap='gray_r')
+ax1.set_axis_off()
+ax1.set_title(f'fraction {fractions[0]}')
+# Show image with highest volume fraction
+ax2 = fig.add_subplot(222)
+ax2.imshow(images[-1], cmap='gray_r')
+ax2.set_axis_off()
+ax2.set_title(f'fraction {fractions[-1]}')
+plt.show()
+
+#####################################################################
+# In the scatterplot, many points seem to be overlapping at low area values.
+# To get a better sense of the distribution, we may want to add some 'jitter'
+# to the visualization. To this end, we use `stripplot` (from `seaborn`, the
+# Python library dedicated to statistical data visualization) with argument
+# `jitter=True`.
+
+sns_fig, sns_ax = plt.subplots()
+sns.stripplot(x='volume fraction', y='area', data=areas, jitter=True,
+              ax=sns_ax)
+# Fix floating point rendering
+sns_ax.set_xticklabels([f'{frac:.2f}' for frac in fractions])
+plt.show()
+
+#####################################################################
+# 3D images
+# =========
+# Doing the same analysis in 3D, we find a much more dramatic behaviour: blobs
+# coalesce into a single, giant piece as the volume fraction crosses ~0.25.
+
+images = [data.binary_blobs(length=128, n_dim=3, volume_fraction=f)
+          for f in fractions]
+
+labeled_images = [measure.label(image) for image in images]
+
+properties = ['label', 'area']
+
+tables = [measure.regionprops_table(image, properties=properties)
+          for image in labeled_images]
+tables = [pd.DataFrame(table) for table in tables]
+
+for fraction, table in zip(fractions, tables):
+    table['volume fraction'] = fraction
+
+blob_volumes = pd.concat(tables, axis=0)
+
+d3_fig, d3_ax = plt.subplots()
```

Comment:
Consider using the same variable names: `fig, ax = plt.subplots()`.

Reply:
I thought it was better to distinguish between all the `fig` and `ax` objects but, now that you point it out, I realize that possible styling additions (by calling methods of these objects) would take place within the same code cell (say, running in a Jupyter notebook) so there should be no confusion...

Reply:
As it goes so far, the examples are made of one or multiple sections (≈ cells), each of which is logically ended up with `plt.plot()` call. So I'd say that your hypothetic scenario is unlikely.

I personally believe that clarity is the metric we should "optimize" the examples for. :)

Reply:
I agree with you. Thanks.

Reply:
Fixed in ae3e04c775acc973b8c59d42b557bda358b6181a.